site/training/assets/training.css

```diff
@@ -0,0 +1,132 @@
+.hero-training {
+  position: relative;
+  margin-top: -35px;
+  padding: 75px;
+  padding-bottom: 50px;
+  color: white;
+  min-height: 250px;
+  z-index: 1; 
+}
+
+.hero-training::before {
+  content: "";
+  /* background: linear-gradient(rgba(8, 62, 68, 1), rgba(8, 62, 68, 0.50)), url('gradient-background.svg'); */
+  background: url('../../assets/img/solutions-hero.png');
+  /* background: url('../gradient-background.svg'); */
+  background-attachment: fixed;
+  background-position: center;
+  background-size: cover;
+  
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: -1; 
+}
+
+a {
+    color: #3E6C69;
+  }
+  
+a:hover {
+    text-decoration: underline 2px solid #042426;
+  }  
+
+#quarto-sidebar.sidebar {
+    background-color: #042426;
+    color: #EAF8FA;
+  }
+
+#quarto-sidebar.menu-text a {
+    color: #EAF8FA;
+  }
+
+.sidebar-item-container .menu-text {
+    color: #EAF8FA;
+}
+
+.sidebar-item-container .sidebar-link:hover .menu-text {
+    color: #DE257E; 
+}
+
+.card {
+    box-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); 
+    border-radius: 5px; 
+    border: 1px solid #3E6C69;
+  }
+  
+.card-title {
+    padding-bottom: 0.2rem;
+    font-size: 100%;
+    font-weight: bold;
+    color: #3E6C69;
+  }
+
+.card:hover {
+    border: 1px solid #042426;
+    background-color: #B5B5B510;
+  }
+
+#listing-sample-notebooks .no-anchor.listing-title a {
+    font-size: 18px; 
+    color: #3E6C69;    
+    font-weight: bold;
+  }
+
+#listing-training-courses .thumbnail-image.card-img {
+    object-fit: cover;      
+    object-position: top;    
+    height: 100%;            
+    width: 100%;             
+  }
+
+#listing-sample-notebooks {
+    background-color: white;
+    padding: 20px;
+    border-radius: 10px;
+    border: 1px solid #042426;
+}
+
+#listing-sample-notebooks .listing-author {
+    color: #083E44; 
+    font-weight: bold;
+  }
+
+.attn {
+    margin-top: 20px;
+    margin-bottom: 10px;
+    border: 2px solid #042426;
+    background-color: #EAF8FA;
+    border-left-width: 0px;
+    border-right-width: 0px;
+    padding-top: 30px;
+    padding-bottom: 20px;
+    padding-left: 30px;
+    padding-right: 40px;
+  }
+  
+.attn h2{
+    color: #042426;
+  }
+
+.preview {
```

Comment:
Need to investigate but I did something to break my `.preview` extension ... It affects all course registration pages. Example:

<img width="920" alt="image" src="https://github.com/user-attachments/assets/758872e7-16bd-44c5-9166-254fbcc73768">

Reply:
It has to be an issue in this PR as Beck's initial PR that my branch is based on does not exhibit this behaviour. But here's a data point: it looks like we're still showing the `.qmd` file in the output when we should be referencing the `.html`:

<img width="1301" alt="image" src="https://github.com/user-attachments/assets/325f8e28-1bd6-471b-aba1-28eedc0b34d8">